pages/components/Badge/badge.css

```diff
@@ -36,3 +36,98 @@
   --bs-bg-opacity: 1;
   background-color: var(--warning-color) !important;
 }
+
+.badge-round {
+  color: var(--white-global);
+  border-radius: 10%;
+  width: 70px;
+  position: inherit;
+  top: 20px;
```

Comment:
any reasons why badge round has to be at this position?

Reply:
made square badges first. 
then round. seperation of concern, no other reason